server/sonar-server/src/main/java/org/sonar/server/rule/ws/SearchAction.java

```diff
@@ -495,7 +454,7 @@ private static SearchWsRequest toSearchWsRequest(Request request) {
       .setActivation(request.paramAsBoolean(PARAM_ACTIVATION))
       .setActiveSeverities(request.paramAsStrings(PARAM_ACTIVE_SEVERITIES))
       .setAsc(request.mandatoryParamAsBoolean(Param.ASCENDING))
-      .setAvailableSince(request.param(PARAM_AVAILABLE_SINCE))
+      // .setAvailableSince(request.param(PARAM_AVAILABLE_SINCE))
```

Comment:
As discussed, I will reactivate this commented line, and remove the copy of this method from RuleQueryFactory